sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -0,0 +1,541 @@
+//
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import {
+  WebResource,
+  ServiceClient,
+  ServiceClientOptions,
+  HttpOperationResponse,
+  ServiceBusSASServiceClientCredentials,
+  AtomXmlOperationSpec,
+  signingPolicy,
+  userAgentPolicy,
+  serviceBusAtomSerializationPolicy,
+  RequestPolicyFactory,
+  ResourceSerializer,
+  URLBuilder
+} from "@azure/core-http";
+
+import { QueueResourceSerializer } from "./serializers/queueResourceSerializer";
+import { TopicResourceSerializer } from "./serializers/topicResourceSerializer";
+import { SubscriptionResourceSerializer } from "./serializers/subscriptionResourceSerializer";
+import { RuleResourceSerializer } from "./serializers/ruleResourceSerializer";
+
+/**
+ * All operations return a `Promise<HttpOperationResponse` object of which the `parsedBody`
+ * is populated with the response in JSON format with following structure:
+ *  {
+ *   response: <actual response>,
+ *   result: <parsed result, if any>,
+ *   error: <error information, if any>
+ *  }
+ */
+export class ServiceBusAtomManagementClient extends ServiceClient {
+  endpoint: any;
+
+  queueResourceSerializer: ResourceSerializer;
+  topicResourceSerializer: ResourceSerializer;
+  subscriptionResourceSerializer: ResourceSerializer;
+  ruleResourceSerializer: ResourceSerializer;
+
+  /**
+   * Initializes a new instance of the ServiceBusManagementClient class.
+   * @param credentials Credentials needed for the client to connect to Azure.
+   * @param options The parameter options
+   */
+  constructor(connectionString: any, options?: ServiceClientOptions) {
```

Comment:
Does the ATOM based management apis only work with Shared Access Keys?
Do they not work with AAD?